example-get-started/analyze.py

```diff
@@ -0,0 +1,42 @@
+import io
```

Comment:
This script is not used in the project, this to generated datasets from the SO raw 1TB dump